<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toSyntaxSetupUI</class>
 <widget class="QWidget" name="toSyntaxSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="OptionsTab">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Syntax highlighting</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="SyntaxHighlighting">
           <item>
            <property name="text">
             <string extracomment="Use Scintilla native SQL Lexer">QsciSQL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="Use vendor specific SQL Lexer (if implemented)">Native</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="EditorTypeLabel">
           <property name="text">
            <string>Editor Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="EditorType">
           <item>
            <property name="text">
             <string>Scintilla</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Em</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="UseMaxTextWidthMarkBool">
           <property name="toolTip">
            <string>Display the line/column marker at the specified position</string>
           </property>
           <property name="text">
            <string>Use &amp;maximum text width mark</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MaxTextWidthMark">
           <property name="toolTip">
            <string>Specify the column (text width) where to show marker</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="CodeCompletionBool">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Activate code completion in SQL editors. Not recommended on slow connections.</string>
         </property>
         <property name="text">
          <string>&amp;Code completion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="KeywordUpper">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Convert all keywords to uppercase when displayingsyntaxhighlighted text.  Only available for monospaced fonts.</string>
         </property>
         <property name="text">
          <string>&amp;Keyword upper</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ObjectNamesUpper">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Table, view, column names etc. are extracted in upper-case</string>
         </property>
         <property name="text">
          <string>Object names upper</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CompletionSort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Sort the available completion rows alphabetically or in the order the are.</string>
         </property>
         <property name="text">
          <string>&amp;Sort completion rows</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="EditorShortcuts">
           <property name="text">
            <string>&amp;Editor shortcuts (TAB replacement)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EditorShortcutsEdit">
           <property name="text">
            <string>E&amp;dit...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="AutoIndentBool">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Indent a new line in an editor to the same column as the previous line</string>
         </property>
         <property name="text">
          <string>&amp;Indent same as previous line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="UseSpacesForIndentBool">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>If set the editor will insert spaces rather than tabs</string>
         </property>
         <property name="text">
          <string>Use only spaces when indenting</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="TabStopLabel">
           <property name="toolTip">
            <string>Number of character width of tabstops</string>
           </property>
           <property name="whatsThis">
            <string>Number of character width of tabstops</string>
           </property>
           <property name="text">
            <string>Tab stop width</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>TabStopInt</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="TabStopInt">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="TextExample">
           <property name="text">
            <string>Text example</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TextExampleChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="CodeExample">
           <property name="text">
            <string>Code example</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CodeExampleChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ResultExample">
           <property name="text">
            <string>List example</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ResultExampleChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Filename extensions</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Extensions"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SyntaxTab">
      <attribute name="title">
       <string>Syntax</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
       <item>
        <widget class="QGroupBox" name="GroupBox2">
         <property name="title">
          <string>Sy&amp;ntax components</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>11</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QListWidget" name="SyntaxComponent"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ExampleColor">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="BGSample">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="PickButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Pick</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="BGColorLabel">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="FontSample">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Sample</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="FontLabel">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QTextBrowser" name="textBrowser">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="openLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="FGColorLabel">
            <property name="text">
             <string>Foreground</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="FGSample">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="toDebugEditor" name="Example" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>toDebugEditor</class>
   <extends>QWidget</extends>
   <header>editor/todebugeditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SyntaxComponent</tabstop>
  <tabstop>PickButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ResultExampleChoose</sender>
   <signal>clicked()</signal>
   <receiver>toSyntaxSetupUI</receiver>
   <slot>selectResultFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CodeExampleChoose</sender>
   <signal>clicked()</signal>
   <receiver>toSyntaxSetupUI</receiver>
   <slot>selectFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>KeywordUpper</sender>
   <signal>toggled(bool)</signal>
   <receiver>toSyntaxSetupUI</receiver>
   <slot>changeUpper(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PickButton</sender>
   <signal>clicked()</signal>
   <receiver>toSyntaxSetupUI</receiver>
   <slot>selectColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SyntaxComponent</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>toSyntaxSetupUI</receiver>
   <slot>changeLine(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CodeCompletionBool</sender>
   <signal>toggled(bool)</signal>
   <receiver>CompletionSort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TextExampleChoose</sender>
   <signal>clicked()</signal>
   <receiver>toSyntaxSetupUI</receiver>
   <slot>selectText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
